integration/spark/src/main/scala/org/apache/spark/sql/execution/command/management/CarbonAddLoadCommand.scala

```diff
@@ -277,6 +277,7 @@ case class CarbonAddLoadCommand(
       operationContext.setProperty(
         carbonTable.getTableUniqueName + "_Segment",
         model.getSegmentId)
+      operationContext.setProperty("isAddLoad", "true")
```

Comment:
Please add comments for the usage of `isAddLoad`

Reply:
added